docker-compose.yml

```diff
@@ -14,7 +14,9 @@ services:
       - AWS_SECRET_KEY=${AWS_SECRET_KEY}
       - ENABLE_ALERTING=1
       - ENABLE_BACKUP_MANAGEMENT=1
-#      - PERCONA_TEST_SAAS_HOST=check.localhost
+#      - PERCONA_TEST_PLATFORM_ADDRESS=https://check.localhost
+#      - PERCONA_TEST_PLATFORM_INSECURE=1
+#      - PERCONA_TEST_PLATFORM_PUBLIC_KEY=<public key>
```

Comment:
nit: it is not clear if it is a PATH to a key

Reply:
It's not a path, it's key itself

Reply:
can you show example of how it is supposed to be used? the key is multiline. the user should cat it can send it?
I think, it can be a potential source of issues. ie when user set value in terminal (escaping and new line issues).

Reply:
It's minisign public key, it's one line, here is example:
https://github.com/percona/pmm-managed/blob/05124a1d4ffae221ddd9d974e2c9f5d926f4afb2/services/checks/checks_test.go#L46


This variable should not be used by anyone outside Percona. We print warnings in logs each time `PERCONA_TEST_*` variable used. It's completely for dev/qa needs.

Reply:
Imagine user creates `.env`, and sets env var this way:
      - PERCONA_TEST_PLATFORM_PUBLIC_KEY=${PERCONA_TEST_PLATFORM_PUBLIC_KEY}

how .env should look like?

<img width="1906" alt="image" src="https://user-images.githubusercontent.com/91831719/172883044-83eff693-e7a8-4834-a9f2-cdcd03d8e643.png">

Reply:
Here is how it will look like for our dev environment:
```
PERCONA_TEST_PLATFORM_PUBLIC_KEY=RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX
```

Reply:
> It's minisign public key, it's one line, here is example:
oh, it is very confusing, because it can be though as https cert
can you pls rename so that it will clear?

Reply:
`PUBLIC_KEY` part is confusing, can we rename it according to it's function? ie `PERCONA_TEST_PLATFORM_ENCRYPTION_KEY`

Reply:
In fact it's not a new variable, it was `PERCONA_TEST_CHECKS_PUBLIC_KEY` for a long time. I renamed it to `PERCONA_TEST_PLATFORM_PUBLIC_KEY` because now it used not only for checks system but also for alerting.

It's _public_ key for signatures verification, not for encryption/decryption. May be `PERCONA_TEST_PLATFORM_VERIFICATION_KEY` will work better. BTW I will create a poll